<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialConfigBox</class>
 <widget class="QWidget" name="SerialConfigBox">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="COMLayout">
        <item>
         <widget class="QLabel" name="COMLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>COM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="COMComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disconnectButton">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <layout class="QHBoxLayout" name="baudrateLayout">
          <item>
           <widget class="QLabel" name="baudrateLabel">
            <property name="text">
             <string>Baudrate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudrateComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dataBitsLayout">
          <item>
           <widget class="QLabel" name="dataBitsLabel">
            <property name="text">
             <string>Data Bits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dataBitsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="parityLayout">
          <item>
           <widget class="QLabel" name="parityLabel">
            <property name="text">
             <string>Parity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="parityComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="stopBitsLayout">
          <item>
           <widget class="QLabel" name="stopBitsLabel">
            <property name="text">
             <string>Stop Bits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stopBitsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="flowControlLayout">
          <item>
           <widget class="QLabel" name="flowControlLabel">
            <property name="text">
             <string>Flow Control</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="flowControlComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="scanButton">
        <property name="text">
         <string> Scan for devices</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
